Presentation.Web/Models/External/V2/ItSystemResponseDTO.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Presentation.Web.Models.External.V2
+{
+    public class ItSystemResponseDTO
+    {
+        /// <summary>
+        /// UUID for IT-System
+        /// </summary>
+        public Guid Uuid { get; set; }
+
+        /// <summary>
+        /// UUID for possible IT-System parent (if any)
+        /// </summary>
+        public Guid? ParentUuid { get; set; }
+        
+        /// <summary>
+        /// Name of IT-System
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Former name of IT-System (if any)
+        /// </summary>
+        public string? FormerName { get; set; }
+
+        /// <summary>
+        /// Description
+        /// </summary>
+        public string Description { get; set; }
+
+        /// <summary>
+        /// Url reference for further information
+        /// </summary>
+        public string UrlReference { get; set; }
+
+        /// <summary>
+        /// UUID for IT-System business type
+        /// </summary>
+        public Guid? BusinessTypeUuid { get; set; }
+
+        /// <summary>
+        /// List of KLE number representations as name and UUID pairs
+        /// </summary>
+        public IEnumerable<IdentityNamePairResponseDTO> KLERepresentations { get; set; }
+
+        /// <summary>
+        /// Active status
+        /// </summary>
+        public bool Deactivated { get; set; }
```

Comment:
mangler "deactivatedreason". Den står i spec

Reply:
Udelod det med vilje ud fra tanken om man kan genaktivere et system (og dermed deaktivere det igen?) I så fald skal feltet hedde LatestDeactivationReason?

Reply:
Ja spørgsmålet er om der er historik (0..*) eller det bare er eet felt. Spørgsmålet er også om værdien af feltet slettes ved genaktivering (afgør hvad det skal hedde) og om det skal vises i UI

Reply:
Har nu talt med Ulla .. de er i det hele taget i tvivl om deaktiverede systemer skal vises, også for rettighedshaver (ikke aktive systemer skal ikke vises for interessenter). Aftalen blev derfor at vi viser (til rettighedshaver) og systemet er deaktiveret eller ej - yderligere historik/grund til deaktivering må de spørge sekretariatet om